Support/kiwix.xcconfig

```diff
@@ -0,0 +1,10 @@
+//  Copyright © 2023 Kiwix. All rights reserved.
+
+// Configuration settings file format documentation can be found at:
+// https://help.apple.com/xcode/#/dev745c5c974
+HAS_LIBRARY = true
```

Comment:
Remove these headers which are in contradction with the GPLv3 license

Reply:
also see dedicated ticket for the headers are: https://github.com/kiwix/apple/issues/575